src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationIntegrationTest.java

```diff
@@ -169,7 +170,35 @@ public void retrieve_all_users_for_an_organisation_with_pagination() {
         assertThat(((List<ProfessionalUsersResponse>) response2.get("users")).size()).isEqualTo(3);
     }
 
-    private void validateUsers(Map<String, Object> response, int expectedUserCount) {
+    @Test
+    public void ac1_find_all_active_users_without_roles_for_an_organisation_should_return_200() {
+        String id = settingUpOrganisation("pui-user-manager");
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisationWithReturnRoles("false", puiCaseManager, id);
+        validateUsers(response, 2, false);
+    }
+
+    @Test
+    public void ac2_find_all_active_users_with_roles_for_an_organisation_should_return_200() {
+        String id = settingUpOrganisation("pui-user-manager");
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisationWithReturnRoles("true", puiCaseManager, id);
+        validateUsers(response, 2, true);
+    }
+
+    @Test
+    public void  ac3_find_all_active_users_with_no_param_given_for_an_organisation_should_return_200() {
+        String id = settingUpOrganisation("pui-user-manager");
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisationWithReturnRoles("", puiCaseManager, id);
+        validateUsers(response, 2, true);
```

Comment:
does this return roles in the response or not?

Reply:
this is fine, i checked it is returning roles when returnRoles parameter is not passed in the API request